packages/cxl-lumo-styles/scss/themes/vaadin-tabs.scss

```diff
@@ -65,3 +65,43 @@ $vaadin-tab-horizontal-padding: 0.75rem;
   margin-right: auto;
   margin-left: auto;
 }
+
+// show horizontal scrollbar
+// needs to refactor, so utilize custom css properties
+:host([orientation="horizontal"][theme~="cxl-jobs-horizontal-slider"]) [part="tabs"] {
+  // override default vaadin-tabs overflow
+  overflow-x: auto;
+
+  &::-webkit-scrollbar {
```

Comment:
We have a dedicated scrollbar styles mixin now, can probably be used here instead of repeating.

https://github.com/conversionxl/aybolit/blob/f17ae7b4e0cd103177f1b19973712601cd6e3835/packages/cxl-lumo-styles/scss/_mixins.scss#L1-L30